t/03.scripts/beekeeper_reset.t

```diff
@@ -134,6 +134,10 @@ foreach my $pipeline_url (@$ehive_test_pipeline_urls) {
     beekeeper($hive_url, ['-discard_ready_jobs'], 'beekeeper.pl -discard_ready_jobs');
     assert_jobs($job_adaptor, [['DONE',1,0],['READY',0,0],['DONE',1,0],['DONE',1,0],['DONE',1,0],['DONE',1,0],['DONE',1,0]] );
 
+    # Reset a job with specific input_id and analysis_pattern
+    beekeeper($hive_url, ['-reset_jobs_for_input_id', '%640%', '-analyses_pattern', 2], 'beekeeper.pl -reset_jobs_for_input_id -analyses_pattern');
```

Comment:
This tests that it doesn't reset a job if the input_id does not match any job. There should be another test that checks that a job gets reset if the input_id does match. For example

`beekeeper($hive_url, ['-reset_jobs_for_input_id', '%4%', '-analyses_pattern', 2], 'beekeeper.pl -reset_jobs_for_input_id -analyses_pattern');`

Which should give expected jobs of
`[['DONE',1,0],['READY',0,0],['DONE',1,0],['DONE',1,0],['DONE',1,0],['DONE',1,0],['READY',1,0]]`